README.md

```diff
@@ -125,6 +125,29 @@ dashboardTemplates:
 
 Note that you can define more than one dashboard under the `dashboardTemplates` field. 
 
+#### Entity TTL
+
+Entities are deleted by default 8 days after the last time they were indexed or sent out a heartbeat. 
+If this doesn't suit your needs you may also specify your desired behavior under the TTL section of the main `definition.yml` file:  
+
+```yaml
+ttl:
+  # The strategy to follow for the entity deletion. Defaults to AUTOMATIC
+  deletionStrategy: AUTOMATIC
+  # The frequency to delete the entity types. Defaults to EIGHT_DAYS
+  deletionFrequency: EIGHT_DAYS
+```
+
+`deletionStrategy` accepts the values `AUTOMATIC` (default) and `MANUAL`. 
+
+If `AUTOMATIC` is used the entity will be automatically deleted after a certain amount of time, defined by the `deletionFrequency`: 
+- `HOURLY`
```

Comment:
Is this TTL actually used by any entity type? If not, I'd remove it as it would be tricky to support it

Reply:
It is used indeed, different entity types have different indexStrategies and deletionFrequencies, the next step after we agree on naming here is to migrate that.

Reply:
I'm referring to the `HOURLY` one specifically. Sorry for the misunderstanding

Reply:
Ah sorry, I misunderstood, I just checked and the only entity using HOURLY is our E2E, so I can remove it from here. Thanks!